packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,18 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// {@macro flutter.material.appbar.automaticallyImplyLeading}
+  ///
+  /// This property is used to configure an [AppBar].
+  bool? automaticallyImplyLeading;
+
+  /// {@template flutter.material.appbar.leadingWidth}
```

Comment:
I think this entire doc string can be replaced with.

```
/// {@macro flutter.material.appbar.leadingWidth}
```

Reply:
i've added your comment the comment line. thank you for fb.